sdk/storage/azblob/zt_access_policy_test.go

```diff
@@ -4,39 +4,43 @@
 package azblob
 
 import (
-	"github.com/stretchr/testify/assert"
 	"strconv"
+	"testing"
 	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
-//nolint
-func (s *azblobUnrecordedTestSuite) TestSetEmptyAccessPolicy() {
-	_assert := assert.New(s.T())
-	testName := s.T().Name()
-	svcClient, err := getServiceClient(nil, testAccountDefault, nil)
-	if err != nil {
-		s.Fail("Unable to fetch service client because " + err.Error())
-	}
+func TestSetEmptyAccessPolicy(t *testing.T) {
+	stop := start(t)
+	defer stop()
+
+	_assert := assert.New(t)
+	testName := t.Name()
+	svcClient, err := createServiceClient(t, testAccountDefault)
+	require.NoError(t, err)
```

Comment:
Any particular reason to use `require` instead of  `_assert` we were using before?

Reply:
The `require` library will fail whereas `_assert` will continue testing and logs that the statement was not what was expected. This causes nil pointer exceptions, panics, and early exits whenever the first test fails. I think it's a huge positive to run all the tests in a suite rather than exit the suite on the first failure.